layouts/partials/tabs.html

```diff
@@ -295,22 +295,72 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
         </li>
 
         <li id="tab-store" class="cd-tabs__panel text-component">
-        <p>Machine Learning ... .</p>
-
-        <p>
-	This tab is work in progress.  Just parking some content here for now.
-        <a href="https://www.tensorflow.org">Tensorflow</a>
-        framework helps to define and run computations involving tensors that have
-        broad applications in Speech and image recognition, Text-based
-        applications, Time-series analysis and Video detection.
-	
-	Other suggestions for Machine Learning:
-
-        Visualization: Yellowbrick, Eli5
-        ML libraries: XGBoost, LightGBM, CatBoost
-        AI libraries: PyTorch, MXNet
-        
-	</p>
+       	   <div class="grid-container">
+	      <div>
+		      <a href="https://ai.googleblog.com/2016/12/open-sourcing-embedding-projector-tool.html"> <img src="images/content_images/ml_img/pytorch-ml-anim.gif" alt="pytorch figure" align="middle" style="border-radius: 10px"></a> 
+              </div>
+	      <div>
+                 <p>
+		 <b>Machine learning</b> (ML) enables computers to learn using
+		    data, without having to be explicitly programmed.
+		    <b>NumPy</b> is the foundation of all data pre-processing
+		    that happens in the implementation of several ML Algorithms.
+		 </p>
+		 <p>
+		    Python’s rich machine language and deep learning ecosystem
+		    provides powerful tools such as
+		    <a href="https://scikit-learn.org/stable/">Scikit-learn</a>
+		    that is built on top of NumPy and
+		    <a href="https://www.scipy.org">SciPy</a> and offers data
+		    mining and analytics using classical ML algorithms.
+		    <a href="">Chainer</a> is another Python based open source
+		    deep learning framework built on top of Numpy and CuPy
+		    Python libraries.
+		 </p>
+	      </div>
+	   </div>
+	   <div>
+	      <div>
+               	 <p>
+		    <a href="https://www.tensorflow.org">Tensorflow’s</a>
+		    deep learning capabilities help to define and run
+		    computations involving tensors. Tensors have broad
+		    applications in Speech and image recognition, Text-based
+		    applications, Time-series analysis and video detection.
+		    <a href="https://keras.io">Keras</a> provides high level
+		    neural network APIs. <a href="https://pytorch.org">PyTorch
```

Comment:
I'd suggest a more equal distribution of text between Tensorflow and PyTorch. The latter is already more popular with researchers than the former. Also JAX deserves a brief mention.

Reply:
And let's mention MXNet here rather than in the next paragraph, that fits better (in scope it's similar to Tensorflow and PyTorch).

Reply:
ok done.

Reply:
Removed logos, edited text as suggested.
<img width="847" alt="Screenshot 2019-12-19 at 11 29 27 AM" src="https://user-images.githubusercontent.com/5890484/71148868-e00f8e80-2252-11ea-808b-74f911dc56da.png">